ocfweb/docs/docs/services/lab/lab-reservation-policy.md

```diff
@@ -0,0 +1,46 @@
+[[!meta title="Lab Reservation Policy"]]
```

Comment:
By convention we only uppercase the first letter of the first word, e.g. "Lab reservation policy"